blog.html

```diff
@@ -0,0 +1,87 @@
+  </head>
+  <body>
+
+
+    <header class="header-images">
+      <div class="background2Box">
+      <img class="logoHeader" src="./images/adoptadog-logo.svg" alt="Company Logo">
+      <h2 class="headertext">Seattle's premiere dog adoption agency</h2>
+      <a href="./checkout.html">Checkout</a>
+    </div>
+          <a class="checkout" href="./checkout.html">Checkout</a>
+    </header>
+
+    <nav>
+      <ul>
+        <li>
+          <a href="./index.html">Home</a>
+        </li>
+        <li>
+          <a href="./dogs.html">Dogs</a>
+        </li>
+        <li>
+          <a href="#">Blog</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main class="grid main-content" id="wrapper">
+      <section>
+        <h2>Adoptadog Blog</h2>
+          <article class="row instructors c-box">
+            <h2 class="hide">Traveling With Your Dogh</h2>
+            <img class="col span_3_of_12" src="./images/blog-1.jpg" alt="Traval with your dogs">
+              <p class="col span_3_of_12 p2">Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. <b>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</b></p>
+
+          </article>
+
+
+          <article class="row instructors c-box">
+            <h2 class="hide">How To Walk Multiple Dogs</h2>
+            <img class="col span_3_of_12" src="./images/blog-2.jpg" alt="dog sitting">
+            <p class="col span_3_of_12 p2">TIduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.
+Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+
+          </article>
+
+          <article class="row instructors c-box">
+            <h2 class="hide">Teach Your Dog To Fetch!</h2>
+            <img src="./images/blog-3.jpg"  class="col span_3_of_12" alt="Teach Your Dog To Fetch!">
```

Comment:
be careful writing your own grid. It can go well, but for things like this might not look good at smaller breakpoints. We'll learn about this on Thursday.